crates/cxx-qt-gen/src/parser/method.rs

```diff
@@ -26,6 +27,7 @@ impl ParsedQInvokableSpecifiers {
             ParsedQInvokableSpecifiers::Final => "cxx_final",
             ParsedQInvokableSpecifiers::Override => "cxx_override",
             ParsedQInvokableSpecifiers::Virtual => "cxx_virtual",
+            ParsedQInvokableSpecifiers::Pure => "cxx_pure",
```

Comment:
can we add a changelog entry and update the book that i mentions these here https://kdab.github.io/cxx-qt/book/bridge/extern_rustqt.html#specifiers